heatmap-service/src/heatmap_response.rs

```diff
@@ -1,18 +1,44 @@
+use chrono::NaiveDate;
 use geojson::FeatureCollection;
 use serde::{Deserialize, Serialize};
 
-use crate::{dataset::Dataset, granule::Granule, heatmap_data::HeatmapData};
+use crate::{granule::Granule, heatmap_data::HeatmapData};
 
 #[derive(Deserialize, Serialize, Debug, PartialEq)]
 pub struct HeatmapResponse {
```

Comment:
this should use the heatmap-api crate version of the struct...... but I can move the code over in my pr once it comes along to prevent having  to do a bunch of manual merging since I make a ton of changes to the server.

Reply:
Yep! I missed switching that, I will leave it alone if you want to deal with it in your PR though!